Update DNS to reflect decision point (#127274)
Azure/azure-content: articles/azure-vmware/native-dns-forward-lookup-zone.md

 
 ### Prerequisite
 
 First, create two /28 subnets to delegate to the Azure DNS Private Resolver service, as an example. They can be named dns-in and dns-out. 
 
 ### Deploy Azure DNS Private Resolver
 
 In your Resource Group, deploy the Private DNS Resolver. 
 
 1. Click create. 
 2. In the Search the Marketplace field, type Private DNS Resolver and click Enter. 
 3. Click Create for the Private DNS Resolver. 
 4. Ensure the Subscription, Resource group, and Region fields are correct. Enter a name and choose your Virtual Network. This network must be the same as the one where you deployed your private cloud; then, click Next: Inbound Endpoints. 

>  5. Click 'Add an Endpoint', enter a name for the Inbound endpoint (e.g., dns-in), select the subnet for the DNS inbound endpoint, and click 'Save'.

Reviewer: ```suggestion
 5. Click 'Add an Endpoint', enter a name for the Inbound endpoint, such as dns-in, and select the subnet for the DNS inbound endpoint, and click 'Save'.
```
Style guide

 6. Click Next: Outbound Endpoints. 

>  7. Click "Add an Endpoint," enter a name for the Outbound endpoint (e.g., dns-out), select the subnet for the DNS outbound endpoint, and click "Save."

Reviewer: ```suggestion
 7. Click "Add an Endpoint," enter a name for the Outbound endpoint, such as dns-out and select the subnet for the DNS outbound endpoint, and click "Save."
```
